oaknational/Oak-Web-Application
# LESQ-36 feat(unit-listing): add optionality options to the unit listing page

## Description

display optionality units on unit listing page

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1741--oak-web-application.netlify.thenational.academy/beta/teachers-2023/programmes/english-secondary-aqa/units
2. Click on _______
3. You should see different optionality units displayed

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
